### DISCUSSION TITLE
LYWSD03MMC not found in Bluetooth after flashing in Zigbee

### DISCUSSION THREAD

**Discussion by Irwine1986** [pvvx/ATC_MiThermometer#424] [Q&A] [answered]
Hi,

I flashed my LYWSD03MMC with the zigbee fw and after that i cant reconnect to the device with bluetooth. It does not appear between the bluetooth devices. The bluetooth icon shows continuously on its display. I tried to reset it, but nothing happens...

What could be the problem?

Thanks!

**Comment by giallu** [comment]
Just happened the same to me, did you find a solution?

For the records, the device works now as Zigbee, everything looks good other than "can't flash it back"

```11:54:56 PM: Searching for devices
11:55:03 PM: Connecting to: LYWSD03MMC
11:55:05 PM: Hardware Revision String: B1.9
11:55:05 PM: Software Revision String: 0130
11:55:05 PM: Firmware Revision String: 1.0.0_0130
11:55:05 PM: Detected Mi device
11:55:05 PM: Connected
11:55:38 PM: Activating now, please wait...
11:55:41 PM: Registration successful
11:55:41 PM: Send Login, please wait...
11:55:42 PM: Received device infos are correct
11:55:42 PM: Login successful
11:56:20 PM: Load firmware file 'ATC_v46.bin'...
11:56:20 PM: File: ATC_v46.bin
11:56:20 PM: File size: 82612 bytes
11:56:20 PM: Count: 5164
11:56:26 PM: Load firmware file 'https://raw.githubusercontent.com/pvvx/ZigbeeTLc/master/bin/Z03MMC_v0118.bin'...
11:56:26 PM: File: https://raw.githubusercontent.com/pvvx/ZigbeeTLc/master/bin/Z03MMC_v0118.bin
11:56:26 PM: File size: 127972 bytes
11:56:26 PM: Count: 7999
11:56:29 PM: Start DFU
11:57:34 PM: Update done after 63.49 seconds
11:57:40 PM: Disconnected.
```

**Comment by giallu** [comment]
> The bluetooth icon shows continuously on its display. I tried to reset it, but nothing happens...

So, according to documentation I found out the BT icon means the device is ready to pair in Zigbee. In fact, it went away when I connected it to my zigbee hub.

> 
> What could be the problem?

I am afraid we made a mistake and flashed a firmware with no support for BT. If this is true I am not sure we can recover the device without a UART programmer...

**Comment by pvvx** [comment, marked as answer]
For the Zigbee option, [Zigbee OTA](https://github.com/pvvx/ZigbeeTLc?tab=readme-ov-file#zigbee-ota) works.

[README](https://github.com/pvvx/ATC_MiThermometer#)
_Support for Zigbee firmware is in the [ZigbeeTLc](https://github.com/pvvx/ZigbeeTLc) repository._

**Reply by giallu** [reply]
Like this? https://github.com/pvvx/ZigbeeTLc/issues/7
